interpreter/cling/lib/Interpreter/MultiplexInterpreterCallbacks.h

```diff
@@ -139,6 +139,22 @@ namespace cling {
       for (auto&& cb : m_Callbacks)
         cb->ReturnedFromUserCode(StateInfo);
     }
+
+    void* LockCompilationDuringUserCodeExecution() override {
+      void* ret = nullptr;
+      for (auto&& cb : m_Callbacks) {
+        if (void* myret = cb->LockCompilationDuringUserCodeExecution()) {
+          assert(!ret && "Multiple state infos are not supported!");
+          ret = myret;
+        }
+      }
+      return ret;
+    }
+
+    void UnlockCompilationDuringUserCodeExecution(void* StateInfo) override {
+      for (auto&& cb : m_Callbacks)
+        cb->UnlockCompilationDuringUserCodeExecution(StateInfo);
```

Comment:
See above.  StateInfo is likely wrong for most of the element of m_Callbacks. (except when there is only zero or one :) )